editoast/src/infra_cache/object_cache.rs

```diff
@@ -1,6 +1,14 @@
 mod buffer_stop_cache;
 mod detector_cache;
 mod electrification_cache;
+mod operational_point_cache;
 
 pub use buffer_stop_cache::BufferStopCache;
 pub use detector_cache::DetectorCache;
+pub use operational_point_cache::OperationalPointCache;
+
+cfg_if! {
+    if #[cfg(test)] {
+        pub use operational_point_cache::OperationalPointPartCache;
+    }
+}
```

Comment:
Just a curiosity, do you know why we do import this one only under testing conditions?